src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingwhitespaceafterasterisk/InputJavadocMissingWhitespaceAfterAsterisk.java

```diff
@@ -0,0 +1,59 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmissingwhitespaceafterasterisk;
+class InputJavadocMissingWhitespaceAfterAsterisk
+{
+    /****
+     *
+     * This is ok.
+     ****/
+    void foo() {}
+
+    /**
+     *  @see Something
+     */
+    void foo1() {}
+
+    /**
+     *Violation.
```

Comment:
please create Input that has javadocs in most possible places (class, methods, fileds, enums, inner classes, ) and make all cases valid. It will be clear definiton of what is expected as correct.
Please create another Input file (class, methods, fields, ..... ) that will contains invalid cases.